galley/pkg/runtime/resource/event.go

```diff
@@ -37,6 +37,10 @@ const (
 	// FullSync indicates that the initial state of the store has been published as a series of Added events.
 	// Events after FullSync are actual change events that the source-store has encountered.
 	FullSync
+
+	// DeletedTypeURL indicates that existing resources for the TypeURL have been deleted
+	// Typically this will be followed by resource additions and FullSync
+	DeletedTypeURL
```

Comment:
This marks a request to delete all resources of the given type? Let's call it `DeleteType`. What's the use case, though? It seems like we would want to avoid dumping all configs of a specific type (it blows up caches, for example).

Reply:
Removed DeletedTypeURL